http/src/test/scala/org/broadinstitute/dsde/workbench/leonardo/http/service/AppServiceInterpSpec.scala

```diff
@@ -1487,12 +1486,14 @@ final class AppServiceInterpSpec extends AnyFlatSpec with LeonardoTestSuite with
     savedDisk.map(_.name) shouldEqual Some(diskName)
   }
 
-  // TODO: Write test with existing disk
```

Comment:
Refactored these tests a bit -- don't think they were quite testing the right behavior before.